server/routes/teachersRoutes.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const router = express.Router();
 const db = require('../data/helpers/teacherHelper');
-const jwtCheck = require('../middleware/authenticate');
+const management = require('../authentication');
```

Comment:
How do we use this, in place of jwtCheck? It's imported, but I don't see it called anywhere.